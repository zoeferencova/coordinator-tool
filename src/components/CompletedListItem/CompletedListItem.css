i {
    font-size: 15px;
    margin: 0 5px;
}

.revert-button {
    font-size: 12px;
}

div .comp-proj-cell {
    text-align: left;
    padding: 10px;
}