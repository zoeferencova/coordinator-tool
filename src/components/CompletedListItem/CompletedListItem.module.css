.revertButton {
    font-size: 12px;
}