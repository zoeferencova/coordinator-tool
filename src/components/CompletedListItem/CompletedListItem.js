import React from 'react';
import './CompletedListItem.css'

export default class CompletedListItem extends React.Component {
    
    render() {
        return (
            <div className="table-row">
                <div className="table-body-cell proj-cell">{this.props.project}</div>
                <div className="table-body-cell adv-cell">{this.props.advisor}</div>
                <div className="table-body-cell">{this.props.pm_name}</div>
                <div className="table-body-cell">{this.props.date_created}</div>
                <div className="table-body-cell">
                    <button>Revert</button>
                </div>
            </div>
        )
    }
}