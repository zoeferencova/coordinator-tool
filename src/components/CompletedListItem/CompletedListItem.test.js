import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import CompletedListItem from './CompletedListItem'

describe(`Completed List Item component`, () => {
  it('renders a .CompletedListItem by default', () => {
    const wrapper = shallow(<CompletedListItem />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})