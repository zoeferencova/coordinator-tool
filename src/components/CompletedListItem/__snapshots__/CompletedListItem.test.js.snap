// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Completed List Item component renders a .CompletedListItem by default 1`] = `
<div
  className="tableRow"
>
  <div
    className="tableBodyCell project"
  />
  <div
    className="tableBodyCell advisor"
  />
  <div
    className="tableBodyCell pm"
  />
  <div
    className="tableBodyCell date hideMobile"
  >
    Invalid Date
  </div>
  <div
    className="tableBodyCell actions"
  >
    <button
      className="revertButton"
      onClick={[Function]}
    >
      Revert
    </button>
  </div>
</div>
`;
